pdoc/html_helpers.py

```diff
@@ -145,18 +147,20 @@ def lookup(module, refname, link_prefix):
             return None, None
     if isinstance(d, pdoc.doc.Module):
         return d.refname, module_url(module, d, link_prefix)
+    suffix = '()' if isinstance(d, pdoc.doc.Function) else ''
+    name = d.qualname if get_qualname else d.name
```

Comment:
Function references are by default labeled with their [full qualified name](https://www.python.org/dev/peps/pep-3155/), e.g. `C.f` or `D.g` in case of methods. Previously, it was the whole refname (`module.submodule.submodule.whole.hierarchy.class.method`). I think this is better, YMMV. Open to discussion.

Before when these helpers were part of the HTML template, it was much easier to override them (and specify `--template-dir`).